packages/code/src/Code.tsx

```diff
@@ -1,29 +1,35 @@
 import React, { useRef, useEffect, useState, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import ClipboardJS from 'clipboard';
+import facepaint from 'facepaint';
+import debounce from 'lodash/debounce';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { useIsomorphicLayoutEffect } from '@leafygreen-ui/hooks';
-import Syntax from './Syntax';
+import { uiColors } from '@leafygreen-ui/palette';
 import { spacing } from '@leafygreen-ui/tokens';
-import { variantColors } from './globalStyles';
-import { Language, CodeProps, Mode } from './types';
 import CheckmarkIcon from '@leafygreen-ui/icon/dist/Checkmark';
 import CopyIcon from '@leafygreen-ui/icon/dist/Copy';
 import IconButton from '@leafygreen-ui/icon-button';
 import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
+import { variantColors } from './globalStyles';
+import { Language, CodeProps, Mode } from './types';
+import Syntax from './Syntax';
 import WindowChrome from './WindowChrome';
-import debounce from 'lodash/debounce';
-import { uiColors } from '@leafygreen-ui/palette';
-import ClipboardJS from 'clipboard';
-import facepaint from 'facepaint';
 
 // We use max-device-width to select specifically for iOS devices
 const mq = facepaint([
```

Comment:
If outside the scope of this PR, we should make a ticket to update this to use our breakpoints token

Reply:
I did a lot of testing for this back when I made this update, and found that the breakpoints tokens didn't target the right devices for what we needed to do here.